frontend/packages/dev-console/src/components/monitoring/MonitoringPage.tsx

```diff
@@ -20,22 +18,15 @@ type MonitoringPageProps = {
   }>;
 };
 
-type StateProps = {
-  canAccess: boolean;
-};
-
-type Props = MonitoringPageProps & StateProps;
-
 const handleNamespaceChange = (newNamespace: string): void => {
   if (newNamespace === ALL_NAMESPACES_KEY) {
     history.push(MONITORING_ALL_NS_PAGE_URI);
   }
 };
 
-export const PageContents: React.FC<Props> = ({ match, canAccess }) => {
+export const PageContents: React.FC<MonitoringPageProps> = ({ match }) => {
   const activeNamespace = match.params.ns;
-  const canAccessPrometheus = canAccess && !!window.SERVER_FLAGS.prometheusBaseURL;
-
+  const canAccessPrometheus = !!window.SERVER_FLAGS.prometheusBaseURL;
```

Comment:
Should this be `prometheusTenancyBaseURL`?

Reply:
`window.SERVER_FLAGS.prometheusBaseURL` / `window.SERVER_FLAGS.prometheusTenancyBaseURL` is constant, so there should be no need for the `React.useMemo` below now.

Reply:
Updated it.